src/activities/guesscount/Guesscount.qml

```diff
@@ -0,0 +1,415 @@
+                                topMargin: 5
+                            }
+
+                            model: availableModes
+                            background: dialogActivityConfig
+                            label: qsTr("Select your mode")
+                        }
+                        Row {
+                            id: labels
+                            spacing: 20
+                            anchors {
+                                top: modeBox.bottom
+                                topMargin: 5
+                            }
+                            visible: modeBox.currentIndex == 0
+                            Repeater {
+                                model: 2
+                                Row {
+                                    spacing: 10
+                                    Rectangle {
+                                        id: label
+                                        width: background.width*0.3
+                                        height: background.height/15
+                                        radius: 20.0;
+                                        color: modelData ? "green" : "red"
+                                        GCText {
+                                            anchors.horizontalCenter: parent.horizontalCenter
+                                            anchors.verticalCenter: parent.verticalCenter
+                                            fontSize: smallSize
+                                            text: modelData ? qsTr("Selected") : qsTr("Not Selected")
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                        Rectangle {
+                            width: parent.width
+                            color: "transparent"
+                            height: parent.height/1.25-labels.height-modeBox.height
+                            anchors {
+                                top: labels.bottom
+                                topMargin: 5
+                            }
+                            ListView {
+                                anchors.fill: parent
+                                visible: modeBox.currentIndex == 0
+                                spacing: 5
+                                model: Activity.numberOfLevel
+                                clip: true
+                                delegate: Admin {
+                                    id: level
+                                    level: modelData
+                                    levelOperators: items.levelArr
+                                    width: background.width
+                                    height: background.height/10
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            onClose: {
```

Comment:
instead of redefining the onClose function, you can use the dataValidationFunc attribute. You can take a look at  balancebox where it's used.